ExtendedDescription.md

```diff
@@ -1,121 +1,154 @@
-## About nelson.social
+# nelson.social
+
+## Server Rules
```

Comment:
The reason the server rules were in a separate file is that, unless something has changed in v4, that section of the `/about` page is generated from the [list of rules in the database](https://nelson.social/admin/rules). So I figured having rest of that /about page content in a separate file makes it easier to "deploy" changes to this copy by copy/pasting the rendered HTML of the `ExtendedDescription.md` from GitHub straight into the `Extended description` field in https://nelson.social/admin/settings/about

What was your thinking about merging it into this file? A way to preview how that whole page looks?

Reply:
Yeah, and Mastodon 4 actually swaps them around: About, then Server Rules, but Hometown didn't take over that change.